files/zh-cn/web/api/window/open/index.md

```diff
@@ -10,27 +10,25 @@ slug: Web/API/Window/open
 ## 语法
 
 ```js
-let windowObjectReference = window.open(strUrl, strWindowName, [
-  strWindowFeatures,
-]);
+open();
+open(url);
+open(url, target);
+open(url, target, windowFeatures);
 ```
 
-> **备注：** *strUrl === *要在新打开的窗口中加载的 URL。
->
-> *strWindowName === *新窗口的名称。
->
-> *strWindowFeatures === *一个可选参数，列出新窗口的特征 (大小，位置，滚动条等) 作为一个{{domxref("DOMString")}}。
-
 ## 参数与返回值
```

Comment:
```suggestion
### 参数
```

Reply:
Hi @btea, we may need to do the full sync here, the content is far different from en-US version.

https://github.com/mdn/content/blob/e34ad429620c8ec7e822e66bc59b941f24e5c91d/files/en-us/web/api/window/open/index.md?plain=1#L22